pos_margin/models/pos_order_line.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2017 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+import openerp.addons.decimal_precision as dp
+
+
+class PosOrderLine(models.Model):
+    _inherit = 'pos.order.line'
+
+    # Columns Section
```

Comment:
Maybe this obvious comments can go away ;)

Reply:
;-) Indeed. My bad.